portfolio/src/main/webapp/script.js

```diff
@@ -23,60 +23,91 @@ function sayHello() {
   });
 }
 
-window.onload = authentication();
+window.onload = init();
+
+function init() {
+  authentication();
+  fetchBlobstoreUrlAndShowForm();
+}
+
+let commentsLimit = 5;
 
 // Retrieves json comments from server 
 function loadComments() {
```

Comment:
This file is starting to get large enough to consider breaking out part of it into another more focused file. See if you can take the Comment logic and move it into a `comment.js`.

Reply:
moved the comment logic out into comment.css